### PR TITLE
fix: the script page style is abnormal.

### PR DESCRIPTION
**Pull request by zhb666**
Issues: https://github.com/Magickbase/ckb-explorer-public-issues/issues/302

### CODE REVIEW

File: src/pages/Script/styles.module.scss [hunk 1]
```diff
@@ -23,30 +23,33 @@
   width: 100%;
```
Comment (Keith-CY) [inline src/pages/Script/styles.module.scss:23]: Some changes are introduced by local linter

Comment (zhb666) [reply src/pages/Script/styles.module.scss:23]: The code has been configured with automatic formatting for preservation. Is the issue here referring to automatic line breaks?

Comment (Keith-CY) [reply src/pages/Script/styles.module.scss:23]: Changes in this file cover `auto blank line insertion` and `different wrap rule` at https://github.com/Magickbase/ckb-explorer-frontend/pull/4/files/1fb945c437c81dc490c4f5956ae011193c16dc13#diff-8e04ae6888bd6cfe2f387c1cd287aec98acf334e382f165e6c6f3513d5301946R48-R52

Are they formatted by the linter defined in the project or the local linter, e.g., installed in IDE.

**Comment by Keith-CY** [comment]
`Capacity Usage` still overflows
image
https://ckb-explorer-git-fork-zhb666-fix-page-style-magickbase.vercel.app/script/0x9bd7e06f3ecf4be0f2fcd2188b23f1b9fcc88e5d4b65a8637b17723bbda3cce8/type/deployed_cells?page=1&size=10

**Comment by zhb666** [comment]
> `Capacity Usage` still overflows image https://ckb-explorer-git-fork-zhb666-fix-page-style-magickbase.vercel.app/script/0x9bd7e06f3ecf4be0f2fcd2188b23f1b9fcc88e5d4b65a8637b17723bbda3cce8/type/deployed_cells?page=1&size=10

has been fixed

File: src/pages/Script/styles.module.scss (hunk 1 above)
Comment (zhb666) [reply src/pages/Script/styles.module.scss]: There was a conflict with the local CSS formatting, but it has been resolved and unified now.

**Review by Keith-CY — APPROVED** [review]

**Comment by FrederLu** [comment]
After switching to mobile or changing the resolution Capacity Usage still overflows
https://ckb-explorer-1l17vozvg-magickbase.vercel.app/script/0x9bd7e06f3ecf4be0f2fcd2188b23f1b9fcc88e5d4b65a8637b17723bbda3cce8/type/deployed_cells?page=1&size=10
image

**Comment by zhb666** [comment]
> After switching to mobile or changing the resolution Capacity Usage still overflows https://ckb-explorer-1l17vozvg-magickbase.vercel.app/script/0x9bd7e06f3ecf4be0f2fcd2188b23f1b9fcc88e5d4b65a8637b17723bbda3cce8/type/deployed_cells?page=1&size=10 image

#### The issue introduced in the previous modification impacted the previously fixed code, but it has now been resolved.
image

**Review by FrederLu — APPROVED** [review]

**Pull request merged**